CONTRIBUTING.md

```diff
@@ -39,9 +39,20 @@ Guidelines for rosdep rules
 ---------------------------
 
  * Native packages are strongly preferred (They are required for packaging and have upgrade and conflict tracking.)
- * Python packages should go in the python.yaml
- * Homebrew into homebrew.yaml
+ * Please use the smallest notation possible and if something is default or not necessary to specify, don't.
+ * Python packages should go in the python.yaml.
+ * Homebrew into osx-homebrew.yaml
  * If a package is only available via pip use the `-pip` extension.
+ * Supported Platforms
+   * Rules can be contributed for any platform. However to be released they must be at least cover the supported platforms in REP 3: http://www.ros.org/reps/rep-0003.html So:
+    * Ubuntu and Debian are top priority.
+    * Fedora and Gentoo have packaging jobs should be best effort.
```

Comment:
This isn't a sentence, maybe you started to say something and changed or there is a word missing?